### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by xiaosi-xu**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,19 +70,35 @@ The repository is structured as follows:
...
+    
+  * **Traveling Salesman Problem**
+
+    This tutorial solves small instances of the famous travelling salesman problem (TSP) using D-Wave devices on Amazon Braket. TSP is an NP-hard problem in combinatorial optimization, asking for the shortest possible route that visits each city exactly once, given a list of cities and the distances between each pair of cities. To solve the problem, cities with distances is mapped to a graph with weighted edges, and a Hamiltonian cycle with the least weight is searched for as the solution.
```
Comment (lundql) [inline README.md:101]: This tutorial solves small instances of the famous travelling salesman problem (TSP) using D-Wave devices on Amazon Braket. TSP is an NP-hard problem in combinatorial optimization. The solution finds the shortest possible route that visits each city exactly once, given a list of cities and the distances between each pair of cities. To solve the problem, cities and distances are mapped to a graph with weighted edges. A solution, when found on that graph, is the Hamiltonian cycle that has the least weight.

**Review by lundql — CHANGES_REQUESTED** [review]
I made some suggestions Some are small things like a missing word. Some are suggestions to break up sentences and thereby add a bit more clarity. Thanks for letting me review this text.

**Comment by xiaosi-xu** [comment]
> I made some suggestions Some are small things like a missing word. Some are suggestions to break up sentences and thereby add a bit more clarity. Thanks for letting me review this text.

Thanks a lot! I have incorporated all the changes.

File: README.md [hunk 2]
```diff
@@ -70,19 +70,35 @@ The repository is structured as follows:
...
   * **Maximum Cut**
   
-    This tutorial solves a small instance of the famous Maximum-Cut (MaxCut) Problem using the D-Wave device on Amazon Braket. The MaxCut problem is one of the most famous NP-hard problems in combinatorial optimization. Given an undirected graph 𝐺(𝑉,𝐸) with a vertex set 𝑉 and an edge set 𝐸, the Max Cut problem seeks to partition 𝑉 into two sets, such that the number of edges between the two sets (considered to be severed by the cut), is as large as possible. Applications can be found in clustering problems for marketing purposes, or for portfolio optimization problems in finance. 
+    This tutorial solves a small instance of the famous maximum cut (MaxCut) problem using a D-Wave device on Amazon Braket. The MaxCut problem is one of the most famous NP-hard problems in combinatorial optimization. Given an undirected graph 𝐺(𝑉,𝐸) with a vertex set 𝑉 and an edge set 𝐸, the MaxCut problem seeks to partition 𝑉 into two sets, such that the number of edges between the two sets (considered to be severed by the cut), is as large as possible. Applications can be found in clustering problems for marketing purposes, or for portfolio optimization problems in finance.
```
Comment (licedric) [inline README.md:81]:
```suggestion
    This tutorial solves a small instance of the famous Maximum Cut (MaxCut) problem using a D-Wave device on Amazon Braket. The MaxCut problem is one of the most famous NP-hard problems in combinatorial optimization. Given an undirected graph 𝐺(𝑉,𝐸) with a vertex set 𝑉 and an edge set 𝐸, the MaxCut problem seeks to partition 𝑉 into two sets, such that the number of edges between the two sets (considered to be severed by the cut) is as large as possible. Applications can be found in clustering problems for marketing purposes, or for portfolio optimization problems in finance. 
```

Not sure why this change was necessary - Max Cut, like most named computational problems, is generally always capitalized in the literature.

File: README.md [hunk 3]
```diff
@@ -70,19 +70,35 @@ The repository is structured as follows:
...
+    This tutorial solves a small instance of the famous maximum cut (MaxCut) problem using a D-Wave device on Amazon Braket. The MaxCut problem is one of the most famous NP-hard problems in combinatorial optimization. Given an undirected graph 𝐺(𝑉,𝐸) with a vertex set 𝑉 and an edge set 𝐸, the MaxCut problem seeks to partition 𝑉 into two sets, such that the number of edges between the two sets (considered to be severed by the cut), is as large as possible. Applications can be found in clustering problems for marketing purposes, or for portfolio optimization problems in finance. 
   
   * **Minimum Vertex**
```
Comment (licedric) [inline README.md:83]: Should this be minimum vertex cover?

File: README.md [hunk 4]
```diff
@@ -70,19 +70,35 @@ The repository is structured as follows:
...
+   
   * **Graph partitioning**
   
-    This tutorial solves a small instance of a graph partitioning problem using the D-Wave device on Amazon Braket. The derivation for this QUBO problem is nicely explained here: https://github.com/dwave-examples/graph-partitioning.
+    This tutorial solves a small instance of a graph partitioning problem using a D-Wave device on Amazon Braket. The derivation for this QUBO problem is nicely explained here: https://github.com/dwave-examples/graph-partitioning.
+
+  * **Factoring**
+  
+    This tutorial shows how to solve a constraint satisfaction problem (CSP) problem, with the example of factoring, using a D-Wave device on Amazon Braket. Particularly, the factoring is expressed as a CSP using Boolean logic operations, and it is converted to a binary quadratic model that can be solved by a D-Wave device.
```
Comment (licedric) [inline README.md:93]:
```suggestion
    This tutorial shows how to solve a constraint satisfaction problem (CSP) problem, with the example of factoring, using a D-Wave device on Amazon Braket. Particularly, factoring is expressed as a CSP using Boolean logic operations, and it is converted to a binary quadratic model that can be solved by a D-Wave device.
```

File: README.md [hunk 5]
```diff
@@ -70,19 +70,35 @@ The repository is structured as follows:
...
+      
+  * **Traveling Salesman Problem**
+
+    This tutorial solves small instances of the famous travelling salesman problem (TSP) using D-Wave devices on Amazon Braket. TSP is an NP-hard problem in combinatorial optimization. The solution finds the shortest possible route that visits each city exactly once, given a list of cities and the distances between each pair of cities. To solve the problem, cities and distances are mapped to a graph with weighted edges. A solution, when found on that graph, is the Hamiltonian cycle that has the least weight.
```
Comment (licedric) [inline README.md:101]:
```suggestion
    This tutorial solves small instances of the famous traveling salesman problem (TSP) using a D-Wave device on Amazon Braket. TSP is an NP-hard problem in combinatorial optimization. The solution finds the shortest possible route that visits each city exactly once, given a list of cities and the distances between each pair of cities. To solve the problem, cities and distances are mapped to a graph with weighted edges. A solution, when found on that graph, is the Hamiltonian cycle that has the least weight.
```

**Review by lundql — APPROVED** [review]

**Pull request merged**